packages/@react-spectrum/overlays/src/Modal.tsx

```diff
@@ -90,8 +91,13 @@ let ModalWrapper = forwardRef(function (props: ModalWrapperProps, ref: RefObject
     otherProps.className
   );
 
+  let viewport = useViewportSize();
+  let style: any = {
+    '--spectrum-visual-viewport-height': viewport.height + 'px'
+  };
```

Comment:
Ahh thats awesome, didn't know we could set up a css variable outside of the css file.